src/Compilers/CSharp/CSharpAnalyzerDriver/CSharpDeclarationComputer.cs

```diff
@@ -125,6 +125,18 @@ private static bool InvalidLevel(int? level)
                         return;
                     }
 
+                case SyntaxKind.ArrowExpressionClause:
```

Comment:
It's somewhat strange that the new shorthand syntax for property declarations has "Expression" in its name instead of "Declaration", but that's obviously not an issue for this PR. (Confused by the name, at first I thought this might be related to pointer dereferencing in unsafe code.)